web3/contract.py

```diff
@@ -17,10 +17,6 @@
     is_text,
     to_tuple,
 )
-from toolz.functoolz import (
```

Comment:
Also replaced imports where toolz was used instead of cytoolz.
Is there any good reason why cytoolz wasn't used?
I am relying on unit tests to tell me if there's something incompatible between cytoolz and toolz.

Reply:
There was a historical issue where `cytoolz.compose` threw an error if you wrapped it in a `functools.partial`.  I think this has been resolved now.

Reply:
Ok, I'll test this `toolz` stuff manually just to be sure!

Reply:
Done! Tested all occurrences of toolz imports manually. Did not find any incompatibility issues!